package.json

```diff
@@ -104,18 +104,18 @@
     "check:eslint": "eslint -v && eslint --max-warnings=0 --ext '.js,.ts' src test",
     "check:tsd": "tsd --version && tsd",
     "check:dts": "./node_modules/typescript/bin/tsc --noEmit mongodb.d.ts && tsd",
-    "check:test": "mocha --file test/tools/runner test/integration",
-    "check:unit": "mocha test/unit/",
+    "check:test": "mocha --config test/mocharc_int.jsonc test/integration",
```

Comment:
the file seems to be `mocha_mongodb.json`

Reply:
fixed